src/download/0.6.0/release-notes.html

```diff
@@ -2444,7 +2444,7 @@ <h1>0.6.0 Release Notes</h1>
     _ = 0xff00_00ff;
     _ = 0b10000000_10101010;
     _ = 0b1000_0000_1010_1010;
-    _ = 0x123_190.109_038_018p102;
+    _ = 0x123_190.109_038_018_102;
```

Comment:
I assumed the "p" was a typo

Reply:
I think it's legit. That literal came from https://github.com/ziglang/zig/issues/504 and `docgen` release-notes actually runs that code and it passes. https://en.cppreference.com/w/cpp/language/floating_literal also shows the same syntax.

Reply:
Ok, I reverted it.